events/testdata/apigw-websocket-request.json

```diff
@@ -97,11 +97,11 @@
     "extendedRequestId": "TWegAcC4EowCHnA=",
     "integrationLatency": "123",
     "messageDirection": "IN",
-    "messageId": null,
```

Comment:
`"messageId": null` should remain in the testdata

Reply:
"messageId" field is not documented for CONNECT eventType.

https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-websocket-api-integration-requests.html#api-gateway-simple-proxy-for-lambda-input-format-websocket

And I couldn't find out a way to keep this field here in the testdata and also to keep the MessageID field of the `APIGatewayWebsocketProxyRequestContext` struct as interface{}.
If I can remove this field from the testdata, the MessageID field can stay as interface{}.

What should I do?